lib/liquid_voting_web/resolvers/voting.ex

```diff
@@ -40,15 +40,18 @@ defmodule LiquidVotingWeb.Resolvers.Voting do
          message: "Could not create vote with given email",
          details: ChangesetErrors.error_details(changeset)}
 
-      {:ok, participant} ->
+      {:ok, _participant} ->
+        %Voting.Participant{uuid: participant_uuid} =
```

Comment:
Is there a benefit from pattern matching for `participant_uuid` here instead of using the returned `participant` in the tuple and retrieving the uuid with `participant.uuid` below?

Reply:
this is the upsert issue. the uuid returned in the resulting struct does not match the uuid of the updated row in the original form (it is a newly generated uuid)

Reply:
ah, ok. I'll have to play with this to make sense of this problem. I haven't seen anything in the code that might cause something like that.

Reply:
I'm not sure, but have a hunch the Ecto.multi operation might be the answer.

Reply:
from our slack investigation: https://github.com/elixir-ecto/ecto/issues/1966#issuecomment-304469636